firebase-auth.html

```diff
@@ -136,6 +136,20 @@
         },
 
         /**
+         * Authenticates a Firebase client using an oauth id_token.
+         *
+         * @return {Promise} Promise that handles success and failure.
+         */
+        signInWithCredential: function(credential) {
+          if (!this.auth) {
+            return Promise.reject('No app configured for firebase-auth!');
+          }
+
+          return this._handleSignIn(this.auth.signInWithCredential(credential));
+        },
+
+
+        /**
```

Comment:
There are two newlines between the comma and the start of the next comment -- the rest of the file has only one newline between. Remove one and I'll be happy to merge :smile: